app/src/controllers/organizations/reducer.js

```diff
@@ -17,11 +17,13 @@
 import { combineReducers } from 'redux';
 import { fetchReducer } from 'controllers/fetch';
 import { loadingReducer } from 'controllers/loading';
+import { alternativePaginationReducer } from 'controllers/pagination';
 import { organizationReducer } from 'controllers/organizations/organization/reducer';
-import { NAMESPACE } from './constants';
+import { initialPaginationState, NAMESPACE } from './constants';
 
 export const organizationsReducer = combineReducers({
   list: fetchReducer(NAMESPACE, { contentPath: 'items' }),
```

Comment:
I guess it's also time to move the `organizations list` store part to the `instance` level controllers.
As we discussed previously, thus we will have the next store structure:
```
-instance
---organizations // (just organizations list that needed for instance's level organizations page)
---allUsers
---etc
organization // (all the things used for a single organization level)
---activeOrganization
---projects
---orgMembers
---etc
```

Reply:
Done.